src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -190,13 +203,15 @@ public void stop() {
     }
 
     public void coast() {
-        driveMotor.setIdleMode(IdleMode.kCoast);
-        turningMotor.setIdleMode(IdleMode.kCoast);
+        int errorCode = driveMotor.setIdleMode(IdleMode.kCoast).value;
+        errorCode += turningMotor.setIdleMode(IdleMode.kCoast).value;
+        if(errorCode != 0){DriverStation.reportError("An Error has occured in SwerveModule.coast() Code:" + REVLibError.fromInt(errorCode).toString(), null);}
     }
 
     public void brake() {
-        driveMotor.setIdleMode(IdleMode.kBrake);
-        turningMotor.setIdleMode(IdleMode.kBrake);
+        int errorCode = driveMotor.setIdleMode(IdleMode.kBrake).value;
+        errorCode += turningMotor.setIdleMode(IdleMode.kBrake).value;
```

Comment:
check drive motor and turning motor separately

Reply:
outdated, resolving